.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Azure IoTEdge PR checklist:
+
+This checklist is used to make sure that common guidelines for a pull request are followed.
+
+### General Guidelines and Best Practices
+- [ ] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
+- [ ] Title of the pull request is clear and informative.
+- [ ] Description of the pull request includes a brief summary of the enhancement or bug fix.
```

Comment:
I love [this](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20). Are "clear and informative" and "brief summary" enough?

Reply:
Yep, I like that link as well :). It is hard to provide that level of guidance in the PR template and not have it become a book. I've added a note at the bottom that describes kodiak's behavior and linked to this article.  Let me know if this addresses what you have in mind.